src/core/ActivityInfoTree.h

```diff
@@ -61,12 +65,15 @@ protected slots:
 private:
     // this is the full activity list, it never changes
     QList<ActivityInfo *> m_menuTreeFull;
+    QList<ActivityInfo *> m_menuTreeFulltemp;
```

Comment:
if it was for optimization, we can remove it along with m_searched and m_searchedtext